images/image-repo-list

```diff
@@ -3,8 +3,5 @@ e2eRegistry: k8sprow.azurecr.io/kubernetes-e2e-test-images
 promoterE2eRegistry: k8sprow.azurecr.io/kubernetes-e2e-test-images
 etcdRegistry: k8sprow.azurecr.io/kubernetes-e2e-test-images
 gcRegistry: k8sprow.azurecr.io/kubernetes-e2e-test-images
-hazelcastRegistry: k8sprow.azurecr.io/kubernetes-e2e-test-images
 PrivateRegistry: e2eteam
```

Comment:
```suggestion
privateRegistry: e2eteam
```

Reply:
I don't think `privateRegistry` is used by any Windows jobs. Could you confirm?

Reply:
We run
 `Kubernetes e2e suite.[sig-node] Container Runtime blackbox test when running a container with a new image should not be able to pull from private registry without secret [NodeConformance]` 
 and
  `Kubernetes e2e suite.[sig-node] Container Runtime blackbox test when running a container with a new image should be able to pull from private registry with secret [NodeConformance]` on windows nodes. 
I think those use the private registry.

Reply:
It uses `gcAuthenticatedRegistry` instead of `privateRegistry`. The only image that is using `privateRegistry` is `AgnhostPrivate`, and I don't think we are testing those cases that use `AgnhostPrivate`

Reply:
Hmm, looks like that test uses the gcAuthenticatedRegistry.

I think this is the only test that uses privateRepositry
https://github.com/kubernetes/kubernetes/blob/ac5f05b3748959dca0eb91007878233c82cf79df/test/e2e/apps/replica_set.go#L102

and I don't see these being run on the GCE test passes like https://testgrid.k8s.io/sig-windows-gce#gce-windows-2019-containerd-master

Reply:
Right, it should have been ``privateRegistry``, not ``PrivateRegistry``.

The private registry is only used for the AgnhostPrivate image (agnhost:2.6) for only 2 tests, one mentioned by Mark, and the other one here:

https://github.com/kubernetes/kubernetes/blob/ac5f05b3748959dca0eb91007878233c82cf79df/test/e2e/apps/rc.go#L71

Those tests didn't run because they're not Conformance (they have ``ginkgo.It`` instead of ``ginkgo.ConformanceIt``), and they are skipped on platforms that are not gce or gke due to:

```
e2eskipper.SkipUnlessProviderIs("gce", "gke")
```

It seems that they were skipped just because it requires private images. Other than that, we should be able to run those tests as well, but at the moment, we don't really have a way to Promote Images in k8s.gcr.io to a private registry that can then be used for testing. At least, not yet.

Nonetheless, we should fix this typo.